data/resource-templates/LDS_Distribution.json

```diff
@@ -2,21 +2,28 @@
     "o:label": "LDS Distribution",
     "o:resource_template_property": [
         {
-            "o:alternate_label": "Inhouds-URL",
-            "o:alternate_comment": "The URL where the distribution can be directly accessed.",
-            "o:is_required": true,
+            "o:alternate_label": "Naam",
```

Comment:
Most of the translations seem English, but the `o:alternate_label` is in Dutch? 

Seems inconsistent

Reply:
Is the choice of the PO